package/boot/uboot-layerscape/files/fsl_ls1046a-frwy-uEnv.txt

```diff
@@ -0,0 +1,9 @@
+fdtaddr=0x8f000000
+loadaddr=0x81000000
+fdt_high=0xffffffffffffffff
+initrd_high=0xffffffffffffffff
```

Comment:
Please add bootm_size to consolidate envs because https://github.com/openwrt/openwrt/commit/65f8d7360c041ed90724bcc97673b30ce704cc33 was merged.

Reply:
Done.